hmftools-build.py

```diff
@@ -0,0 +1,67 @@
+import re
+import subprocess
+import sys
+import xml.etree.ElementTree as ET
+
+# Check if a semver version is included as argument.
+if len(sys.argv) != 2:
+    print(f'Invalid arguments. Usage: {sys.argv[0]} semver-version')
```

Comment:
Put semver-version in `<semver-version>`, convention for printing help message on the command line.